hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-cop.tex

```diff
@@ -1,4 +1,12 @@
-% Unicode (UTF-8) patterns generated by copthyph-utf8 script
+% title: Unicode (UTF-8) patterns generated by copthyph-utf8 script
+% copyright: Probably Claudio Beccari
```

Comment:
Probably???

Reply:
For sure according to https://ctan.org/tex-archive/language/coptic/cbcoptic